Gemfile

```diff
@@ -54,23 +58,23 @@ gem 'nokogumbo', '1.4.9'
 gem 'rest-client', '~> 1.8.0', :require => 'rest_client'
 gem 'resque', '>=1.14.0'
 gem 'resque_mailer'
-gem 'resque-scheduler', :require => 'resque_scheduler'
+gem 'resque-scheduler'
 #gem 'daemon-spawn', :require => 'daemon_spawn'
 gem 'tire'
 gem 'elasticsearch'
 gem 'aws-sdk'
 gem 'css_parser'
 
 gem 'cocaine'
-gem 'paperclip'
+gem 'paperclip', '4.3.6'
 
 # for looking up image dimensions quickly
 gem 'fastimage'
 
 # Gems for authentication
 gem 'devise', '~> 3.0'   # Lock on version 3 until we update to Rails 4
```

Comment:
Oh no worries - I noticed 3.5 or so works with 4.0 so I bumped it as far as was compatible on its own branch. I'll let it run and see how the tests go. If nothing major breaks, we can do a minor bump with 4.0 if that works.